Misc-Tools.lua

```diff
@@ -117,66 +161,102 @@ local function OnCreateMove(pCmd)
 
     -- Random Fakelag
     if mRandLag:GetValue() == true then
-        if pLocal:IsAlive() and pCmd.command_number % 20 == 0 then
-            local randValue = math.random(120, 315)
+        flTimer = flTimer +1
+        if (flTimer >= mRandLagValue:GetValue()) then
+            flTimer = 0
+            local randValue = math.random(mRandLagMin:GetValue(), mRandLagMax:GetValue())
             gui.SetValue("fake lag value", randValue)
         end
     end
 
+    -- Random Ping
+    if mRandPing:GetValue() == true then
+        prTimer = prTimer +1
+        if (prTimer >= mRandPingValue:GetValue() * 66) then
+            prTimer = 0
+            local prActive = gui.GetValue("ping reducer")
+            if (prActive == 0) then
+                gui.SetValue("ping reducer", 1)
+            elseif (prActive == 1) then
+                gui.SetValue("ping reducer", 0) 
+            end
+        end
+    end
+
     -- Anti RTD
     if mRemovals:IsSelected("RTD Effects") then
         if CurrentRTD == "Cursed" then
             pCmd:SetForwardMove(pCmd:GetForwardMove() * (-1))
             pCmd:SetSideMove(pCmd:GetSideMove() * (-1))
         elseif CurrentRTD == "Drugged" or CurrentRTD == "Bad Sauce" then
-            SetOptionTemp("norecoil", 1)
+            --SetOptionTemp("norecoil", 1) --can get you banned in community servers 🤔
```

Comment:
I think i should remove this whole RTD section anyways and only keep the screen effect removals

Reply:
`return` works

Reply:
That's your call. Personally, I do appreciate the reverse direction keys.